homeassistant/components/demo/alarm_control_panel.py

```diff
@@ -5,7 +5,7 @@
 https://home-assistant.io/components/demo/
 """
 import datetime
-from homeassistant.components.alarm_control_panel import manual
+from homeassistant.components.manual.alarm_control_panel import ManualAlarm
```

Comment:
Will the component be named `manual`? Is that intended?

Reply:
that is the name of the existing platform. We have quite a few names for the same thing: `generic`, `manual`, `generic_thermostat`. I did not want to do breaking changes in here.

Reply:
Yeah. `manual` is just so very generic for a component name. :smile:

We shouldn't change it here though.